db/python/tables/analysis.py

```diff
@@ -347,7 +294,7 @@ async def get_latest_complete_analysis_for_type(
 
         _query = f"""
 SELECT a.id as id, a.type as type, a.status as status,
-        a.output as output, a_sg.sample_id as sample_id,
+        a.output as output, a_sg.sequencing_group_id as sequencing_group_id,
```

Comment:
Same agg treatment here might avoid copying the a.meta multiple time in this query?